src/main/resources/assets/styles/_profiledBox.scss

```diff
@@ -1,3 +1,45 @@
 .part-profiled-box {
   display: flex;
+
+  .left-orientation {
+
+    .card-content.with-image {
+
+      .preambleText {
+        display: -webkit-box;
+        -webkit-line-clamp: 5;
```

Comment:
Would we not get better cross-browser support with something like [Clamp.js](https://clamp.js) ? Even better something like [React Line Ellipsis](https://xiaody.github.io/react-lines-ellipsis/) which allows us much more control and reuse ? (also responsive by design)